views/treetest/edit.ejs

```diff
@@ -55,9 +55,77 @@
 				</div>
 				<div class='col-sm-8'>
 					<div class="well">
-						<p><label for="tasks">Tasks</label>
-						<br>One task per line (task numbers will be added automatically).</p>
-						<textarea style='width:100%' name="tasks" rows="15" cols="30" id="tasks"><% for(var j=0; j<singleStudy.data.tasks.length; j++) {%><%= singleStudy.data.tasks[j]+'\n' %><% } %></textarea>
+						<p>
+							<label for="tasks">Tasks</label>
+							<br>One task per line (task numbers will be added automatically).
+						</p>		
+						<p>			
+							<button id="newTask" type="button" class="btn btn-secondary" data-toggle="tooltip" data-placement="right" title="Adds a new task">
```

Comment:
`data-toggle="tooltip"` is a bootstrap attribute for displaying a tooltip when hovered over the element. 
`data-placement="right"` controls where the tooltip will be displayed relative to the parent element. 

Bootstrap tooltip requires to have a `title` attribute to tell what text to be displayed.

https://getbootstrap.com/docs/4.0/components/tooltips/

Reply:
That makes sense. I still think...

```html
<button>
    <!-- icon -->
    New task
</button>
```

...is a more robust and more usable solution than...

```html
<button data-bs-toggle="tooltip" data-bs-placement="right" title="Adds a new task">
    <!-- icon -->
</button>
```

because the text is always visible instead of only when you hover over it, and because all the functionality is in the HTML, instead of depending on Bootstrap's JavaScript. But I don't think it's very important -- up to you if you want to change it :)

Reply:
Did we use tooltip to keep it consistent with the rest of the UI? Otherwise I would agree with Joe.